docs/User Manual.rst

```diff
@@ -768,6 +774,7 @@ Example 1: ::
       8016.03    0.018535464; }
       }
       water { temp 273;
+      rgb (0 0 200); // Blue colour is used for water in plots
```

Comment:
I'm actually not a big fan of the comments in the manual, the text should be enough to explain everything.

Reply:
Ok. I will remove it then ;-) 
I think you are right. It is needless repetition of the data in the paragraph, which has a good chance to go out of date.